Boundary Docs (#2925)
hashicorp/web-unified-docs: content/boundary/v1.0.x/content/docs/targets/connections/index.mdx

---
layout: docs
page_title: Boundary connection workflows
description: >-
  Compare Boundary's target connection workflows, including transparent sessions, connect helpers, the -exec flag, and SSH ProxyCommand, to pick the right fit.
# START AUTO GENERATED METADATA, DO NOT EDIT
created_at: 2025-11-18T17:04:41-05:00
last_modified: 2026-01-26T13:07:13-05:00
# END AUTO GENERATED METADATA
---

# Boundary connection workflows

> Boundary offers several workflows for connecting to a target, including the `boundary connect` command, transparent sessions, connect helpers, the `-exec` flag, SSH ProxyCommand, and multi-hop sessions. Each workflow trades setup effort against how transparent the connection feels to end users. This page compares the workflows so you can choose one; for setup steps, refer to each workflow's own page.

Reviewer: ```suggestion
Boundary offers several workflows for connecting to a target, including the `boundary connect` command, transparent sessions, connect helpers, the `-exec` flag, SSH ProxyCommand, and multi-hop sessions. Each workflow trades setup effort against how transparent the connection feels to end users. For configuration steps, refer to each workflow's own page.
```
